# Improvements in Metabolic Syndrome Criteria by Xanthohumol Derivatives are Linked to Changes in Fecal Microbiota in a Murine Metabolic Syndrome Model 
===
![](https://img.shields.io/badge/PhD-project-hotpink.svg?style=flat)

We previously showed that two hydrogenated xanthohumol (XN) derivatives, α, β-dihydro-XN (DXN) and tetrahydro-XN (TXN), improved several criteria of metabolic syndrome (MetS), a critical risk factor of cardiovascular disease 
(CVD), in a murine model for diet-induced MetS. To evaluate whether MetS improvements are linked to changes in the microbiome, we fed C57BL/6J male mice a high-fat diet (HFD) or HFD containing XN, DXN or TXN for 13 weeks with a 
submaximal dose (for XN) of 30 mg/kg diet/day. We sequenced 16S rRNA gene and measured bile acids (BAs) in fecal samples and genes associated with epithelial barrier, inflammation, as well as bile acid metabolism, in samples from 
colon, white adipose tissue, and liver, respectively. Both XN-derivatives significantly decreased the percentages of Bacteroidetes and Tenericutes, and TXN altered BA composition and decreased gene expression associated with 
chronic inflammation. Transkingdom network analysis allowed us to link changes in fecal Marvinbryantia, Romboutsia and other undefined genera from the Lachnospiraceae family with changes in BA composition and improvements in MetS 
indicators, which will be the focus of our future studies. In conclusion, microbiota changes may in part explain diet (TXN)-induced improvements in MetS criteria and CVD risk.
===

## Overview

```text

2017_XN_16S/
├── README.md
├── results/
│   ├── analyses/
│   │   ├── Figure1/
│	│	│	├── ASV_taxonomy_subset.txt
│	│	│	├── ASVs_counts.txt
│	│	│	├── Figure1_final.Rmd
│	│	│	└── sampledata.txt
│   │   ├── Figure2/
│	│	│	├── ASV_taxonomy_subset.txt
│	│	│	├── ASVs_counts.txt
│	│	│	├── Figure2.Rmd
│	│	│	├── TableSS2.txt
│	│	│	└── sampledata.txt			
│   │   ├── Figure3/
│	│	│	├── ASVs_counts_gg.txt
│	│	│	├── ASVs_taxonomy_gg.txt
│	│	│	├── Figure3.Rmd
│	│	│	├── phyloseq_wtree.rds
│	│	│	├── sampledata.txt
│	│	│	└── seqtabNoC.rds
│   │   ├── Figure4/
│	│	│	├── ASVs_counts_gg.txt
│	│	│	├── ASVs_taxonomy_gg.txt
│	│	│	├── Figure4.Rmd
│	│	│	├── phyloseq.rds
│	│	│	├── posnegc05.txt
│	│	│	├── sampledata.txt
│	│	│	└── sigtab05.txt
│   │   ├── Figure5/
│	│	│	├── Figure5.svg
│	│	│	├── fecal_BA3.txt
│	│	│	└── figure5.Rmd
│   │   ├── Figure6/
│	│	│	├── Figure6.png
│	│	│	├── figure6.Rmd
│	│	│	└── genes.txt
│   │   └── Figure7/
│	│		├── Results.xlsx
│	│		├── net.cys
│	│		├── net.png
│	│		└── network.cys
│   └── Figures/
│       ├── Figure1.png
│       ├── Figure2.png
│       ├── Figure3.png
│       ├── Figure4.png
│       ├── Figure5.png
│       ├── Figure6.png
│       └── Figure7.png
├── data/
│	├── raw/
│	├── DADA2/
│	└── references/
│
└── submission/
	├── study.pdf      # Final manuscript
	├── references.bib # BibTeX formatted references
	└── XXXX.csl       # csl file to format references for journal XXX

```
